source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -88,8 +88,37 @@ title: Verify the data encryption key was created
 ref: verify-data-key-created
```

Comment:
nit: title case on the step title (Missed this in the unification pass)

**Verify that the Data Encryption Key was Created**

Reply:
Thanks, will fix.